independent-projects/qute/core/src/main/java/io/quarkus/qute/ValueResolvers.java

```diff
@@ -108,6 +109,25 @@ public boolean appliesTo(EvalContext context) {
         };
     }
 
+    /**
+     * Return an empty list if the base object is null or {@link Result#NOT_FOUND}.
+     */
+    public static ValueResolver orEmpty() {
+        CompletionStage<Object> empty = CompletableFuture.completedFuture(Collections.emptyList());
```

Comment:
I'm not a reactive guy but can you make this static or not?

Reply:
I don't think it's necessary. In 99% of cases a single instance of the resolver will be used for the application.